ase/src/main/java/com/ase/ase/rest/controller/BasicCovidController.java

```diff
@@ -29,42 +29,59 @@
                         casesTimelineRepository.findAllBy(provinceId)));
     }
 
-
     @CrossOrigin
-    @GetMapping(value = "/hospital/{province-id}", produces = "application/json")
+    @GetMapping(value = "/new-cases/", produces = "application/json")
```

Comment:
the last / should be removed everywhere. ONLY /new-cases

Reply:
If the last / is removed the REST call doesn't work I tried it.

Reply:
I also tried it and it works because it must work because it is a standard that request parameters are passed behind a ? which is immediately after the path without a / in between. The request should look like localhost:8080/daily/new-cases?area=1,2

Reply:
Check https://www.baeldung.com/spring-request-param - there it also works this way